python/pyspark/sql/dataframe.py

```diff
@@ -1273,6 +1273,41 @@ def limit(self, num: int) -> "DataFrame":
         jdf = self._jdf.limit(num)
         return DataFrame(jdf, self.sparkSession)
 
+    def offset(self, num: int) -> "DataFrame":
```

Comment:
Should also add to the Python API reference .rst file I believe.

Reply:
good catch! thanks!